javascript/Makefile

```diff
@@ -0,0 +1,14 @@
+version:
+	@echo $(shell npm view . version)
+
+version-set:
+	@npm version $(NEW_VERSION)
+
+release-prepare: version-set
+	@git add package*.*
+	@git commit -m "Prepare to release v$(NEW_VERSION)"
```

Comment:
Commiting should problebly be done after all changes have been made in all languages.

Reply:
Agreed. Will re-work the recipe in this target.